hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestOzoneRpcClientAbstract.java

```diff
@@ -3753,4 +3757,65 @@ public void testOverWriteKeyWithAndWithOutVersioning() throws Exception {
     createRequiredForVersioningTest(volumeName, bucketName, keyName, true);
     checkExceptedResultForVersioningTest(volumeName, bucketName, keyName, 2);
   }
+
+  @Test
+  public void testSetECReplicationConfigOnBucket()
+      throws IOException {
+    String volumeName = UUID.randomUUID().toString();
+    store.createVolume(volumeName);
+    OzoneVolume volume = store.getVolume(volumeName);
+    OzoneBucket bucket = getBucket(volume);
+    Assert.assertEquals(bucket.getName(), bucket.getName());
```

Comment:
`Assert.assertEquals(bucket.getName(), bucket.getName());`

I don't think this assert is needed as its just checking the same variable equals itself.